great_expectations/datasource/fluent/spark_azure_blob_storage_datasource.py

```diff
@@ -120,6 +123,8 @@ def add_csv_asset(
         name: str,
         batching_regex: Union[re.Pattern, str],
         container: str,
+        header: bool = False,
+        infer_schema: bool = False,
```

Comment:
`non-blocking`
This probably belongs in a different PR but...
I would expect that the `spark` `read_csv` equivalent also takes a lot more arguments than just these 2??

Reply:
@Kilo59 Yes, I would like to collaborate with you on this -- as well as the following two items:
* More reader types for `SparkDatasource` (it is capable of much more than `CSV`);
* Making `add_<type>_asset()` for `SparkDatasource` more elegant.

Thank you.